tests/features/common/kie-workbench-common.feature

```diff
@@ -1,6 +1,15 @@
 @rhdm-7/rhdm-decisioncentral-rhel8 @rhpam-7/rhpam-businesscentral-rhel8
 Feature: Decision/Business Central common features
 
+  Scenario: Web console is available
+    When container is ready
+    Then check that page is served
+      | property             | value        |
+      | port                 | 8080         |
+      | path                 | /kie-wb.jsp  |
+      | expected_status_code | 200          |
+      | wait                 | 120          |
+
```

Comment:
The same test in two places? [duplicate](https://github.com/jboss-container-images/jboss-kie-modules/pull/391/files#diff-77ee78b8923963f9b5e2d2ea63bdbb85870e13e025a4b58fee7a7f2c14279e84R4). In this the image `@rhpam-7/rhpam-businesscentral-monitoring-rhel8` is also present, duplicate could be removed?

Reply:
the tests are the same but target images different, probably better keep them separated.

Reply:
Okay :)